### ISSUE TITLE
Unable to gradle sync the module gradle project when creating a new module using create-expo-module

### ISSUE THREAD

**Issue by fobos531** [expo/expo#20308]
### Summary

Basically, what I'm trying to do is develop a new native module according to the Expo Modules API docs. Everything goes good, the example app's android gradle project syncs just fine, but the place where I get stuck is making changes to the actual module, which requires me to open it in Android Studio and sync the module's gradle project.

The error I get is: 
```
A problem occurred evaluating root project 'android'.
> Project with path ':expo-modules-core' could not be found in root project 'android'.
```

And the line where gradle build fails is:
```
  def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")

```
I'm assuming that's originating from [here](42cb535a4aa28182437640c79f8/packages/expo/android/build.gradle#L39).  I've double checked that my Android dev env is set up correctly, both according to official React Native docs as well as CONTRIBUTING guidelines outlined in this repository. I've tried clearing the entire gradle cache and completely reinstalling Android Studio but the issue still persists. 

Any clues on what might be causing this?

### What platform(s) does this occur on?

Android

### Environment

```
jakovglavina@jakovs-macbook-pro-3 my-module % npx expo-env-info

  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.0.1
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.14.0 - ~/.asdf/installs/nodejs/16.14.0/bin/node
      Yarn: 1.22.19 - ~/.asdf/installs/nodejs/16.14.0/.npm/bin/yarn
      npm: 9.1.3 - ~/.asdf/plugins/nodejs/shims/npm
      Watchman: 2022.11.28.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.1, iOS 16.1, macOS 13.0, tvOS 16.1, watchOS 9.1
    IDEs:
      Android Studio: 2021.3 AI-213.7172.25.2113.9123335
      Xcode: 14.1/14B47b - /usr/bin/xcodebuild
    npmGlobalPackages:
      eas-cli: 2.8.0
      expo-cli: 6.0.8
    Expo Workflow: bare
```

### Minimal reproducible example

https://github.com/fobos531/my-module

**Comment by lukmccall** [comment]
Hi @fobos531, if I understand you correctly, you're trying to open the `android` folder inside of the Android Studio. You shouldn't do that. To edit module code, you still have to open the whole project. There you'll be able to find the module code that can be safely modified. 
![image](https://user-images.githubusercontent.com/9578601/205940836-93da5a02-56a0-4785-90b0-67903201a713.png)

The module on its own doesn't know the location of the `expo-modules-core` or the react native. That information is injected when the project is loading.

**Comment by jaakkonakaza** [comment]
I'm having the same problem, even when opening the root directory instead of the `android` folder in Android Studio. How is the injection of the information you mentioned supposed to work? Is there any documentation on how to get Android Studio to work with native modules?

**Comment by jaakkonakaza** [comment]
The solution was to open `examples/android`

**Comment by fobos531** [comment]
@lukmccall In my case the issue seemed to be due to the fact that I installed node and related tooling using asdf, and it seems in that case Expo's scripts don't play nicely when syncing the android project. When I tried installing node via using nvm, I was able to successfully sync the project. See here for more context about using Expo with asdf: https://twitter.com/ospfranco/status/1623586905244807170

Maybe something can be done to alleviate this issue.
